jobs/generation/Utilities.groovy

```diff
@@ -637,7 +637,9 @@ class Utilities {
             '\'type_traits\' file not found', // This is here for certain flavors of clang on Ubuntu, which can exhibit odd errors.
             '\'typeinfo\' file not found', // This is here for certain flavors of clang on Ubuntu, which can exhibit odd errors.
             'Only AMD64 and I386 are supported', // Appears to be a flaky CMAKE failure
-            'java.util.concurrent.ExecutionException: Invalid object ID'
+            'java.util.concurrent.ExecutionException: Invalid object ID',
+            '.*XmlException.', // This is here until NuGet cache corruption issue is root caused and fixed.
```

Comment:
Thank you both for the inputs. I've removed the 1st regex and only left the 2nd one.